ddev/src/ddev/cli/validate/labeler.py

```diff
@@ -28,8 +28,10 @@ def labeler(app: Application, sync: bool):
         return
 
     valid_integrations = dict.fromkeys(i.name for i in app.repo.integrations.iter("all"))
-    # Remove this when we remove the `datadog_checks_tests_helper` package
-    valid_integrations['datadog_checks_tests_helper'] = None
+
+    include = set(app.repo.config.get('/overrides/validate/labeler/include', []))
```

Comment:
can we use `.` syntax here or is only path-like supported?

I ask because right now we define them using TOML's nested table syntax which uses the dot and it would make it easier at least for me to have similar access syntax.

Reply:
It's not with the implementation we have, we load the TOML file into a [jsonpointer](https://pypi.org/project/jsonpointer/) file